boot/bootutil/src/swap_nsib.c

```diff
@@ -62,7 +62,7 @@ void nsib_swap_run(struct boot_loader_state *state, struct boot_status *bs)
     rc = swap_scramble_trailer_sectors(state, fap_sec);
     assert(rc == 0);
```

Comment:
remove fixup! and put nrf-squash! on new line with the commit that needs to be squashed